pytroll_collectors/s3stalker.py

```diff
@@ -93,18 +185,31 @@ def set_last_fetch(timestamp):
     DatetimeHolder.last_fetch = timestamp
 
 
-def publish_new_files(bucket, config):
+def get_last_fetch():
+    """Get the last fetch time."""
+    return DatetimeHolder.last_fetch
+
+
+def create_messages_for_recent_files(bucket, config, time_back):
```

Comment:
Why does `time_back` need to be passed here? it is available in the config, right?